pandas/core/generic.py

```diff
@@ -10307,7 +10301,7 @@ def _doc_parms(cls):
 
 Returns
 -------
-%(name1)s or %(name2)s (if level specified)
+%(name1)s or %(name2)s (if level specified)\
```

Comment:
Hmm what failure was this causing? Seems strange to be required on this line but not on subsequent substitutions.

Reply:
Quite a few of them:

```
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Series.kurt: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10026;code=GL03;]pandas.Series.mad: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Series.mean: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Series.median: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Series.skew: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Series.kurtosis: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10067;code=GL03;]pandas.Series.compound: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Panel.mean: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Panel.median: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Panel.skew: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=None;linenumber=None;code=GL03;]pandas.core.groupby.DataFrameGroupBy.mad: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=None;linenumber=None;code=GL03;]pandas.core.groupby.DataFrameGroupBy.skew: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10067;code=GL03;]pandas.DataFrame.compound: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.DataFrame.kurt: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.DataFrame.kurtosis: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10026;code=GL03;]pandas.DataFrame.mad: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.DataFrame.mean: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.DataFrame.median: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.DataFrame.skew: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Series.ptp: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10067;code=GL03;]pandas.Panel.compound: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Panel.kurt: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10969;code=GL03;]pandas.Panel.kurtosis: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=10026;code=GL03;]pandas.Panel.mad: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
```

I believe these errors are due to these functions lacking both `see also` and `examples` sections. Alternatively we could modify the `Aggregate` decorator to ignore these empty strings.

Reply:
OK thanks. So I guess the solution here would be to fix all of the See Also's to not begin with a newline, though probably best served as a follow up PR